src/utils/backend/database.ts

```diff
@@ -32,6 +32,11 @@ export function db2Contact(
     },
     value: contact.value,
     type: contact.type,
+    includes: {
+      students: contact.include_students!,
+      parents: contact.include_parents!,
+      teachers: contact.include_teachers!,
```

Comment:
Don’t overuse `!`. It’s like a middle finger to the compiler. Like an “I’m 100% sure this is not null why are you saying it is” to the compiler. I don’t know why you’re using it here so I can’t really comment on the validity of this so do check why and get back to me.